### PR TITLE
Update doc, add server env setup batch file

### PR DESCRIPTION
**Pull request by lingtangraster**
@dengding @hanhu7 
Please review, thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,9 +31,15 @@ This repository is served as a harbor to provide guidance on deep learning pytho
...
    
-   - For ArcGIS Enterprise customers, the deep learning libraries need to be installed to the directory 
-   *C:\ArcGIS\Server\framework\runtime\ArcGIS\bin\Python* on each raster analytics server machine using command lines.
-   Here is an example to install tensorflow:
+   - For ArcGIS Enterprise users, the deep learning libraries need to be installed to the python environment directory: 
+   *C:\ArcGIS\Server\framework\runtime\ArcGIS\bin\Python* on each raster analytics server machine. 
+   The batch file *[env_setup_server.bat](env_setup_server.bat)* at root directory is used to automate the environment setup process. 
+   Note: use it with caution, by default batch installation will take 5 GB disk space on C: drive. To execute the batch file, 
+   right click the batch file and run as administrator. This batch file takes less than 5 minutes to run on the author's test machine.
+   Once finish the deep learning python environment setup, the users need to restart the ArcGIS server in Services Manager.
```
Comment (hanhu7) [inline README.md]:
```suggestion
   Once finishing the deep learning python environment setup, the users need to restart the ArcGIS server in Services Manager. 
```

File: README.md [hunk 2]
```diff
@@ -31,9 +31,15 @@ This repository is served as a harbor to provide guidance on deep learning pytho
    Note: use it with caution, by default batch installation will take 5 GB disk space on C: drive. To execute the batch file, 
    right click the batch file and run as administrator. This batch file takes less than 5 minutes to run on the author's test machine. 
    
-   - For ArcGIS Enterprise customers, the deep learning libraries need to be installed to the directory 
-   *C:\ArcGIS\Server\framework\runtime\ArcGIS\bin\Python* on each raster analytics server machine using command lines.
-   Here is an example to install tensorflow:
+   - For ArcGIS Enterprise users, the deep learning libraries need to be installed to the python environment directory:
```
Comment (hanhu7) [inline README.md]:
```suggestion
   - For ArcGIS Enterprise users, the deep learning libraries need to be installed to the python environment: 
```

File: env_setup_server.bat [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+CALL "%SCRIPTS_DIR%conda.exe" create --name %ENV_NAME% --clone arcgispro-py3
+CALL "%SCRIPTS_DIR%activate.bat" %ENV_NAME%
+
+ECHO INFO: Install tensorflow-gpu, keras, and scikit-image ...
+CALL "%SCRIPTS_DIR%conda.exe" install -c anaconda %TF_PACKAGE% keras scikit-image --yes
+
+ECHO INFO: Install cntk-gpu, pytorch, torchvision, and Pillow ...
+CALL "%ENV_DIR%%ENV_NAME%\Scripts\pip.exe" install %CTNK_PACKAGE% %PYTORCH% torchvision Pillow
+
+ECHO INFO: Upgrading h5py library to latest version ...
```
Comment (dengding) [inline env_setup_server.bat]: If this is to solve the crash while importing tensorflow, there is no need to upgrade hdf5 in ArcGIS 10.7 AO11 server python env.

Comment (lingtangraster) [reply env_setup_server.bat]: thanks for the reminder.

Comment (dengding) [reply env_setup_server.bat]: Another thing is: we should not recommend user to install all dl framework packages into one single virtual environment. Can we separate this batch file into multiple files each of which is for one single deep learning framework?

Comment (lingtangraster) [reply env_setup_server.bat]: done

**Pull request merged**